src/v2/spec.yaml

```diff
@@ -1317,6 +1317,48 @@ components:
           example: true
         action:
           $ref: "#/components/schemas/ValidatedFrameAction"
+    FramesResponse:
+      type: object
+      required:
+        - frames
+      properties:
+        frames:
+          type: array
+          items:
+            type: string
+    TotalInteractorsResponse:
```

Comment:
only keep the response type definitions, don't define the objects inside the response. allows us to move faster